lint.sh

```diff
@@ -6,7 +6,7 @@ flake8
     --jobs=4                                                                                                \
     --max-line-length=120                                                                                   \
     --ignore=E124,E126,E128,E131,E156,E201,E221,E222,E241,E265,E271,E272,E701,F405,E501,W503                \
-    --per-file-ignores=users/common/strings.py:E501
```

Comment:
1. Why you are deleting this file from flake ignores?
2. Please explain why do you add new one to ignore and what do you want to ignore in this file.

Reply:
1. Well, it's funny: I tried to just ADD another file to this rule, but I couldn't figure out the proper syntax to separate the files.
I mean it was supposed to be the newline character, but I couldn't get it to work.
But I checked it and `users/common/strings.py:E501` is not raised anymore, most likely due to some refactoring along the way.
So since it's useless, I just replaced it...

2. `F403 'from behave import *' used; unable to detect undefined names` - so same as before, so up for debate.
`F811 redefinition of unused 'step_impl' from line 10` - so again, that's how `behave` is expected to work.